<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vkapiClass</class>
 <widget class="QMainWindow" name="vkapiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vkapi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="LogButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="QListWidget" name="messagesText">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vkapi.qrc"/>
 </resources>
 <connections/>
</ui>
